src/main/java/com/crud/library/controller/BookController.java

```diff
@@ -0,0 +1,44 @@
+package com.crud.library.controller;
+
+
+import com.crud.library.domain.BookDto;
+import com.crud.library.mapper.BookMapper;
+import com.crud.library.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequestMapping("/v1/library/book")
+public class BookController {
+    private BookMapper bookMapper;
+    private BookService bookService;
+
+    @Autowired
+    public BookController(BookMapper bookMapper, BookService bookService) {
+        this.bookMapper = bookMapper;
+        this.bookService = bookService;
+    }
+
+    @GetMapping(value = "getBooks")
```

Comment:
te nazwy nie są zgodne z zasadami resta, zrób to porządnie :) `/books`, `/books/{id}` itd